spec/features/proposals_spec.rb

```diff
@@ -212,4 +212,26 @@
       expect(page.find('#event_submission_text').value).to eq(event_type.submission_instructions)
     end
   end
+
+  context 'as a user, looking at a conference with scheduled events' do
+    before(:each) do
+      @user = create(:user)
+      @program = conference.program
+      @selected_schedule = create(:schedule, program: @program)
+      @program.update_attributes!(selected_schedule: @selected_schedule)
+      @scheduled_event1 = create(:event, program: @program, state: 'confirmed', abstract: '`markdown`')
+      @event_schedule1 = create(:event_schedule, event: @scheduled_event1, schedule: @selected_schedule, start_time: conference.start_hour + 1.hour)
+    end
+
+    scenario 'for a scheduled event, can add an event to google calendar if signed in', feature: true, js: true do
```

Comment:
The `js: true` is unnecessary, as this test does not rely on JavaScript. Unnecessarily starting the JavaScript framework to run a test slows down the test suite overall.